packages/core/src/redocly/index.ts

```diff
@@ -1,58 +1,78 @@
 import { existsSync, readFileSync, writeFileSync, unlinkSync } from 'fs';
 import { resolve } from 'path';
 import { homedir } from 'os';
-import { yellow, red, green, gray } from 'colorette';
+import { red, green, gray } from 'colorette';
 import { RegistryApi } from './registry-api';
+import { DOMAINS, Region } from '../config/config';
 
 const TOKEN_FILENAME = '.redocly-config.json';
+const DEFAULT_REGION = 'us';
+const DEFAULT_DOMAIN = 'redoc.ly';
 
 export class RedoclyClient {
-  private accessToken: string | undefined;
+  private accessTokens: { us?: string; eu?: string; } | undefined;
+  private domain: string | undefined;
+  private region: Region;
   registryApi: RegistryApi;
 
-  constructor() {
-    this.loadToken();
-    this.registryApi = new RegistryApi(this.accessToken);
+  constructor(region?: Region) {
+    this.region = region || DEFAULT_REGION;
+    this.loadTokens();
+    this.setDomain(region);
```

Comment:
Let's get rid of this extra setter, it is not used anywhere else:

```suggestion
    this.domain = (region
      ? DOMAINS[region]
      : process.env.REDOCLY_DOMAIN || DOMAINS[DEFAULT_REGION]) || DEFAULT_DOMAIN;
```

We can also get rid of getter (if we add `|| DEFAULT_DOMAIN;` here